wagtail/admin/views/pages/create.py

```diff
@@ -182,13 +182,46 @@ def get_view_draft_message_button(self):
     def get_view_live_message_button(self):
         return messages.button(self.page.url, _("View live"), new_window=False)
 
+    def set_default_privacy_setting(self):
+        # privacy setting options ['public',"logged_in", "shared_password", "user_groups"]
```

Comment:
Issue: These should match the existing restriction names (`BaseViewRestriction.RESTRICTION_CHOICES`).